demo/test.html

```diff
@@ -1,204 +1,287 @@
 <html>
-  <head>
-    <title>Radar | Test SDK Script</title>
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
-    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha256-3edrmyuQ0w65f8gfBsqowzjJe2iM6n0nKciPUp8y+7E=" crossorigin="anonymous"></script>
 
-    <script src="../dist/radar.js"></script>
+<head>
```

Comment:
I don't think this is a crazy formatting change - basically, it looks like the `<head>` tag is on the same level as the parent `<html>` tag, when it should be 2 spaces over. Should be fine after that as this mostly looks like whitespace changes

Reply:
bumping this - did you have a chance to try tabbing over everything within `<head>` and `<body>` 2 spaces?